source/extensions/filters/http/cache/http_cache_utils.cc

```diff
@@ -182,6 +182,31 @@ SystemTime HttpCacheUtils::httpTime(const Http::HeaderEntry* header_entry) {
   }
   return {};
 }
+
+absl::optional<uint64_t> HttpCacheUtils::readAndRemoveLeadingDigits(absl::string_view& str) {
+  uint64_t val = 0;
+  uint32_t bytes_consumed = 0;
+
+  for (const char cur : str) {
+    if (cur < '0' || cur > '9') {
```

Comment:
Unfortunately I can't find any alternatives to do this (and get the digit) without custom parsing here :/ My only suggestion is to replace bits with utilities, like the digit check with `absl::ascii_isdigit`.